src/cromshell/utilities/HTTPUtils.py

```diff
@@ -0,0 +1,27 @@
+import logging
+import sys
+
+import requests
+
+from cromshell.utilities import IOUtils
+
+LOGGER = logging.getLogger(__name__)
+
+
+def assert_can_communicate_with_server(cromshell_url):
+    """Check Connection with Cromwell Server"""
+
+    try:
+        curl_out = requests.get("{}/api/workflows/v1/backends".format(cromshell_url))
+    except ConnectionError:
+        LOGGER.error("Failed to connect to %s", cromshell_url)
+        sys.exit()
+    except requests.exceptions.RequestException:
+        raise Exception("Failed to connect to {}".format(cromshell_url)) from None
```

Comment:
F-string.  Probably better to log an error here and exit gracefully than print a stack trace to the user.

Reply:
Added a Log error, and kept the exception but removed the `from None` as suggest Louis